### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bmunkholm**
Updated links to point to MongoDB (both for Product and Code of Conduct)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Realm Studio is the tool, any developer or system administrator would use when building and maintaining their app built
 on the Realm Mobile Platform.
 
-Visit https://realm.io/products/realm-studio/ for more information on Realm Studio.
+Visit https://docs.mongodb.com/realm/studio/ for more information on Realm Studio.
```
Comment (kraenhansen) [inline README.md:6]: I'm I missing something or doesn't the new page contain links to download Studio for the supported platforms?

**Comment by kraenhansen** [comment]
You probably wanted to merge this into `channel/major-11` since that's the default branch now.

**Comment by bmunkholm** [comment]
Ha! There you go, and I thought I was able to make a simple change to the Readme 🤣 - I should know better....

File: README.md (hunk 1 above)
Comment (bmunkholm) [reply README.md:6]: @cbush ?

Comment (bmunkholm) [reply README.md:6]: @kraenhansen It does, you just need to hit the "Install" section: https://docs.mongodb.com/realm/studio/install/

Comment (kraenhansen) [reply README.md:6]: Ahh .. I was on my phone, where I apparently have to perform an unintuitive horizontal scroll to get to the link.

Screenshot_20211123-103808
Screenshot_20211123-103813

**Pull request merged**